debian/source/lintian-overrides

```diff
@@ -2,5 +2,8 @@
 ubuntu-advantage-tools: missing-build-dependency-for-dh_-command dh_systemd_start => dh-systemd
 ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd => dh-systemd
 
+# Lintian doesn't see the second debhelper dependency with >13.3 on jammy++
+ubuntu-advantage-tools: missing-build-dependency-for-dh-addon systemd *debhelper* (>= 9.20160709~)*
+
 # Lintian doesn't like mentioning riscv64 for older go package
```

Comment:
Question: do we need this override? we do not go anymore do we?

Reply:
I've created a follow up issue to look at all warnings and also remove that line here #2608

Reply:
This PR is just a minimal diff to override the error for release 28

Reply:
aha nice good good